ijson/backends/_yajl2.c

```diff
@@ -0,0 +1,942 @@
+/*
+ * C extension to bind yajl to ijson
+ *
+ * Contributed by Rodrigo Tobar <rtobar@icrar.org>
+ *
+ * ICRAR - International Centre for Radio Astronomy Research
+ * (c) UWA - The University of Western Australia, 2016
+ * Copyright by UWA (in the framework of the ICRAR)
+ * All rights reserved
```

Comment:
I'm not sure it's compatible with the BSD license of ijson. The copyright statement itself is fine of course, since you're the author, but "All rights reserved" specifically contradicts the explicit grant of the BSD license to do pretty much anything. Would UWA agree to license this work under BSD, do you think?